contributing-docs/11_provider_packages.rst

```diff
@@ -174,13 +174,9 @@ there where you should add and remove dependencies for providers (following by r
 ``update-providers-dependencies`` pre-commit to synchronize the dependencies with ``pyproject.toml``
 of Airflow).
 
-The old ``provider.yaml`` file is compliant with the schema that is available in
+The ``provider.yaml`` file is compliant with the new schema that is available in
```

Comment:
"new" is realtive. Right after merge this is out-dated. It is the "current" structure. I'd propose to remove the term "new".

Reply:
Indeed. Missed it :)